ptsprojects/zephyr/l2cap.py

```diff
@@ -146,6 +147,40 @@ def test_cases(pts):
                            TestFunc(stack.l2cap_init, psm_authentication_required, le_initial_mtu),
                            TestFunc(btp.l2cap_le_listen, psm_authentication_required)]
 
+    pre_conditions_keysize = [TestFunc(btp.core_reg_svc_gap),
```

Comment:
Why not use a common pre_conditions list and combine it with some specific list for a set of test cases?

`pre_conditions_key_size = pre_conditions + [... something specific to a set of test cases ...]`

Reply:
Ok, because pre_conditions_auth existed before, I added additional commit that simplifies this